pytesmo/validation_framework/adapters.py

```diff
@@ -34,9 +35,39 @@
 import operator
 from pytesmo.time_series.anomaly import calc_anomaly
 from pytesmo.time_series.anomaly import calc_climatology
+from pandas import DataFrame
 
 
-class MaskingAdapter(object):
+class BasicAdapter(object):
+    """
+    Base class for other adapters that works around the ascat reader not
+    returning a DataFrame. Also removes unnecessary timezone information in data.
+    """
+    def __init__(self, cls):
+        self.cls = cls
+
+    def __get_dataframe(self, data):
+        if ((not isinstance(data, DataFrame)) and (hasattr(data, 'data')) and (isinstance(data.data, DataFrame))):
+            data = data.data
+        return data
+
+    def __drop_tz_info(self, data):
+        if data.index.tz is not None:
+            warnings.warn('Dropping timezone information for data')
```

Comment:
Should we put in what the timezone was?

Reply:
i mean stating the timezone that has been dropped, i.e. UTC. May be a clearer message to the user. For example if they didn't know the timezones on the data going in, it would now tell them and may help them resolve errors more easily. Only a suggestion.